DataFormats/TrackerRecHit2D/interface/FastTrackerRecHit.h

```diff
@@ -49,7 +49,7 @@ class FastTrackerRecHit : public BaseTrackerRecHit
     
     /// destructor
     ///
-    ~FastTrackerRecHit() {}
+    virtual ~FastTrackerRecHit() {}
```

Comment:
Hi Vincenzo

On the internet I read

"Deleting a derived class object using a pointer to a base class that has a non-virtual destructor results in undefined behavior."

Is that not true (anymore)?